Solutions/Azure Activity/Analytic Rules/SubscriptionMigration.yaml

```diff
@@ -0,0 +1,62 @@
+id: 48c026d8-7f36-4a95-9568-6f1420d66e37
+kind: Scheduled
+name: Subscription moved to another tenant
+description: |
+  'This detection uses AzureActivity logs (Security category) to identify when a subscription is moved to another tenant.
+  A threat actor may move a subscription into their own tenant to circumvent local resource deployment and logging policies.
+  Once moved, threat actors may deploy resources and perform malicious activities such as crypto mining.
+  This is a technique known as "subscription hijacking". More information can be found here: https://techcommunity.microsoft.com/t5/microsoft-365-defender-blog/hunt-for-compromised-azure-subscriptions-using-microsoft/ba-p/3607121'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+queryPeriod: 20m
+queryFrequency: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1496
+query: |
+  let queryFrequency = 5m;
+  let eventCapture = "moved from tenant ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}) to tenant ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})";
+  AzureActivity
+  | where ingestion_time() > ago(queryFrequency)
+  | where CategoryValue =~ "Security"
+  | where OperationNameValue =~ "Microsoft.Subscription/updateTenant/action"
```

Comment:
@ccmsft In the logs I had, 'Properties' field had this message and Properties_d was coming as null. Can you use following code after line 28 to use first non null value between Properties and Properties_d?

| extend Properties_d = coalesce(todynamic(Properties), Properties_d)

Reply:
Thanks for the recommendation! I used parse_json() in place of todynamic() as the docs indicate that [todynamic is deprecated alias](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/parsejsonfunction#:~:text=Deprecated%20aliases%3A%20parsejson()%2C%20toobject()%2C%20todynamic()).

Reply:
great